content/en/observability_pipelines/setup/datadog_with_archiving.md

```diff
@@ -0,0 +1,529 @@
+      max_size: 51539607552
+  ## This sink writes logs to an S3 bucket of your choice, in a format that Datadog
+  ## can rehydrate from. You need to fill in the ${DD_ARCHIVES_BUCKET}
+  ## and ${DD_ARCHIVES_REGION} parameters to fit your S3 configuration.
+  datadog_archives:
+    type: datadog_archives
+    inputs:
+      - logs_finish_ddtags
+    service: aws_s3
+    bucket: ${DD_ARCHIVES_BUCKET}
+    aws_s3:
+      storage_class: "STANDARD"
+      region: "${DD_ARCHIVES_REGION}"
+EOT
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+### Load balancing
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+Production-oriented setup is not included in the Docker instructions. Instead, refer to your company's standards for load balancing in containerized environments. If you are testing on your local machine, configuring a load balancer is unnecessary.
+{{% /tab %}}
+{{% tab "AWS EKS" %}}
+Use the load balancers provided by your cloud provider.
+They adjust based on autoscaling events that the default Helm setup is configured for. The load balancers are internal-facing,
+so they are only accessible inside your network.
+
+Use the load balancer URL given to you by Helm when you configure the Datadog Agent.
+
+NLBs provisioned by the [AWS Load Balancer Controller][1] are used.
+
+See [Capacity Planning and Scaling][2] for load balancer recommendations when scaling the Worker.
+#### Cross-availability-zone load balancing
+The provided Helm configuration tries to simplify load balancing, but you must take into consideration the potential price implications of cross-AZ traffic. Wherever possible, the samples try to avoid creating situations where multiple cross-AZ hops can happen.
+
+The sample configurations do not enable the cross-zone load balancing feature available in this controller. To enable it, add the following annotation to the `service` block:
+
+```
+service.beta.kubernetes.io/aws-load-balancer-attributes: load_balancing.cross_zone.enabled=true
+```
+
+See [AWS Load Balancer Controller][3] for more details.
+
+[1]: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/
+[2]: /observability_pipelines/architecture/capacity_planning_scaling/
+[3]: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/service/annotations/#load-balancer-attributes
+{{% /tab %}}
+
+{{% tab "APT-based Linux" %}}
+No built-in support for load-balancing is provided, given the single-machine nature of the installation. You will need to provision your own load balancers using whatever your company's standard is.
+{{% /tab %}}
+{{% tab "RPM-based Linux" %}}
+No built-in support for load-balancing is provided, given the single-machine nature of the installation. You will need to provision your own load balancers using whatever your company's standard is.
+{{% /tab %}}
+{{% tab "Terraform (AWS)" %}}
+An NLB is provisioned by the Terraform module, and provisioned to point at the instances. Its DNS address is returned in the `lb-dns` output in Terraform.
```

Comment:
I'm not sure if this reword makes more or less sense, but tried to make it in the active voice:
```suggestion
The Terraform module provisions an NLB to point at the instances. The DNS address is returned in the `lb-dns` output in Terraform.
```